config/gasSchedule.go

```diff
@@ -9,6 +9,8 @@ import (
 
 const GasValueForTests = 1
 
+const isNegativeNumber = 1
```

Comment:
Is this the result of a signum()? It seems strange to have 1 represent negatives. Is this intentional?

Reply:
We can use only uint64 as the sign of a coefficient (on the gas schedule). It is intentional, 0 = unsigned, 1 = signed.